apps/organisations/views.py

```diff
@@ -282,13 +288,16 @@ def generate_image(self, data):
             description, font=fontsm)
 
         # add text using width to center
-        image.text(((sharepic_format['img_min_width'] - title_width) / 2,
-                    sharepic_format['title_y']),
-                   data['title'], fill=(0, 0, 0), font=font)
+        image.text(
+            ((sharepic_format['img_min_width'] - title_width) / 2,
+                sharepic_format['title_y']),
+            title,
+            fill=(0, 0, 0),
+            font=font)
```

Comment:
still not pretty but maybe a bit better?

Reply:
Yes, that's easier to read!